### PR TITLE
Allow any LUYA admin version

### PR DESCRIPTION
**Pull request by nadar**
As with release of LUYA admin 3.0 this throws a composer update error.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	},
 	"require": {
 		"luyadev/luya-core" : "^1.0",
-		"luyadev/luya-module-cms": "^1.0 | ^2.0",
+		"luyadev/luya-module-cms": "*",
```
Comment (cebe) [inline composer.json]: > Allow any LUYA **admin** version

but you made a change for cms module?

How about breaking changes? Setting a `*` might result in composer installing a combination that is incompatible.

Comment (cebe) [inline composer.json]:
```suggestion
		"luyadev/luya-module-cms": "^1.0 | ^2.0 | ^3.0",
```

Comment (nadar) [reply composer.json]: sure, this works as well. We just have to send a PR on next release. If there is a BC break i would have to send a PR with adjustments anyhow, so the `*` would be fine i thought.

Comment (nadar) [reply composer.json]: sure, this works as well. We just have to send a PR on next release. If there is a BC break i would have to send a PR with adjustments anyhow, so the * would be fine i thought.

**Review by cebe — APPROVED** [review]

**Comment by cebe** [comment]
Thank you!

**Pull request merged**